revert_deploy.yml

```diff
@@ -0,0 +1,55 @@
+###
+#  Set to previous deploy in case of an error or problem.
+#
+# Example usage:
+# ansible-playbook -e host_group=prosody_qa revert_deployment.yml
+#
+# The script will ask for confirmation before making any breaking changes.
+# ctrl+c and then 'a 'will break execution, enter or ctrl+c and then 'c' will
+# proceed.
+#
+# NOTE: You will need the password file loaded for any secrets encrypted in
+# the vaults used by the host_group specified.
+###
+
+- hosts: '{{ host_group }}'
+  connection: ssh
+  remote_user: deploy
+  environment:
+    PATH: '{{ path }}'
+    LD_LIBRARY_PATH: '{{ ld_library_path }}'
+    PYTHONPATH: '{{ python_path }}'
+  tasks:
+  - name: Pause and let user make sure command looks right.
+    pause:
+      prompt: >
+               You are about to revert {{ host_group | upper }}.
+               Press enter to proceed or ctrl+c then a to abort.
+  - name: Check that previous symlink exists.
+    stat:
+      path: "{{ install_root }}/previous"
+    register: previous
+  - fail:
+      msg: "Previous symlink does not exist."
```

Comment:
Suggested wording: no previous deploy available (or something more like that)

(User shouldn't care about implementation details like symlink)